src/configure

```diff
@@ -27,7 +27,7 @@ CONFIGURE_VERSION=2
 
 function rel2abs {
   if [ ! -z "$1" ]; then
-    local retval=`cd $1 2>/dev/null && pwd || exit 1`
```

Comment:
The intention was to pass a directory to configure using --prefix=PREFIX, but that the directory may not already exist. I used the existing read_dirname function so as not to duplicate functionality, but discovered that it doesn't return the correct path if it doesn't already exist (which readlink -f) does.

Perhaps it would be ok if the prefix had to already exist, and then we don't have to make any changes to this function.